examples/03-hoisting.md

```diff
@@ -0,0 +1,47 @@
+# Hoisting
+
+Module declarations are hoisted, and can be imported before their declaration:
```

Comment:
this may answer/contradict my earlier question/comment - if they're hoisted to the top of the block, can you reference them in an expression before they're defined also, like `var`?

Reply:
I pushed [`eb53890` (#21)](https://github.com/tc39/proposal-module-fragments/pull/21/commits/eb5389017e16c02930870e83b8391d6407ee258b) to clarify it: yes you can use them before their definition, but it's more like `function` than like `var` (because their value is already present).